app/package.json

```diff
@@ -3,7 +3,7 @@
   "productName": "GitHub Desktop",
   "bundleID": "com.github.GitHubClient",
   "companyName": "GitHub, Inc.",
-  "version": "1.5.0-beta4",
+  "version": "1.5.0-beta5",
```

Comment:
This is probably just my own ignorance, but just checking whether this is correct or whether it should be 1.5.0 (without the beta) for the prod release.

Reply:
This is correct, and Central would complain loudly without a new version to use for the release. I'll push a commit to this branch to make that clear.